plugin/prestashop/webpay/README.md

```diff
@@ -21,18 +21,24 @@
 
 ___
 
-<aside class="notice">
-Estás viendo la <strong>nueva documentación REST</strong> de este plugin. Si quieres ver referencia de la versión anterior
-(SOAP) haz [click aquí](/plugin/prestashop/webpay-soap)
-</aside>
-
 <h1 class="toc-ignore">Webpay REST Prestashop</h1>
 <h1 style="display: none;">Webpay REST</h1>
 
 ## Descripción
 
 Este plugin oficial ha sido creado para que puedas integrar Webpay fácilmente en tu comercio, basado en Prestashop.
 
+Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (las que te solicita certificados y una llave privada), revisa este video para actualizarte.
```

Comment:
replace => solicita / solicitan

Reply:
Thanks! For future reference, you can suggest a change directly in this comment box: 
![image](https://user-images.githubusercontent.com/1103494/103377427-27fbff00-4abe-11eb-87d9-e7dcb41ba820.png)

:)

Reply:
```suggestion
Estás viendo la nueva documentación REST de este plugin. Si aún estás usando la versión anterior SOAP (la que te solicita certificados y una llave privada), revisa este video para actualizarte.
```